src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -566,3 +566,66 @@ const evidenceTypesDescription = (disabilityName) => {
 export const getEvidenceTypesDescription = (form, index) => {
   return evidenceTypesDescription(getDiagnosticCodeName(form.disabilities[index].diagnosticCode));
 };
+
+export const GetStartedMessage = ({ checkDisabilityStatus, isVerified }) => {
```

Comment:
These might make sense to pull out into their own helper file. It would reduce the pre-form JS bundle size if nothing else, since we wouldn't have to import all the dependencies for the other helpers.

Reply:
Good idea, @cvalarida,  new helper file set up.